samples/benchmarks/coremark/Kconfig

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "CoreMark sample"
+
+config APP_MODE_FLASH_AND_RUN
+	bool "Run CoreMark benchmark on start up"
+	default y if SOC_NRF54H20_CPUPPR
+	help
+	  If enabled, CoreMark will start execution immediately after CPU start up.
+	  Otherwise, it will wait button to be pressed.
```

Comment:
Actually with the current implementation the option also disables button and led (I would say it's not expected  so we should either change or document it)